src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java

```diff
@@ -10,18 +10,17 @@
 import javax.annotation.Nonnull;
 import java.util.Optional;
 
-@SuppressWarnings("NullableProblems")
```

Comment:
why is this removed

Reply:
On all the other classes the null problems were just ignored so I didnt bother, ill add it back